README.md

```diff
@@ -1,75 +1,85 @@
+|                 | Project Info                                                 |
+| --------------- | ------------------------------------------------------------ |
+| License:        | Apache License, Version 2.0                                  |
+| Build:          | Gradle                                                       |
+| Documentation:  | http://feedhenry.org/                                        |
+| Issue tracker:  | https://issues.jboss.org/browse/FH                           |
+| Mailing lists:  | [feedhenry-dev](http://feedhenry-dev.2363497.n4.nabble.com/) |
 
-## Build
+## Building
 
-### Prereqs
+```shell
+./gradlew build
+```
 
-* [Java 7](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
-* [Maven 3.1.1](http://maven.apache.org/)
-* Latest [Android SDK](https://developer.android.com/sdk/index.html) and [Platform version](http://developer.android.com/tools/revisions/platforms.html)
-* Latest [Maven Android SDK Deployer](https://github.com/mosabua/maven-android-sdk-deployer)
-* Set ```ANDROID_HOME``` environment variable
+## Usage
 
-### Building
+There are two supported ways of developing apps using Feedhenry for Android: Android Studio and Maven.
 
-This library is built as an aar project using Maven, but Google does not ship all the required libraries to Maven Central. You must locally deploy them using the [maven-android-sdk-deployer](https://github.com/mosabua/maven-android-sdk-deployer).
+### Android Studio
 
-Run the following commands outside the FeedHenry Android SDK directory to avoid polluting the repo:
+Add to your application's `build.gradle` file
 
+```groovy
+dependencies {
+  compile 'com.feedhenry:fh-android-sdk:3.3.0'
+}
 ```
-git clone git://github.com/mosabua/maven-android-sdk-deployer.git
-cd $PWD/maven-android-sdk-deployer/platforms/android-23
-mvn install -N --quiet
-cd -
-cd $PWD/maven-android-sdk-deployer/repositories/google-m2repository
-mvn install -N --quiet
-```
-
-Now let's build the library.
 
-In the base repo directory:
-
-```
-cd $PWD/fh-android-sdk
-mvn clean package
+### Maven
+
+Include the following dependencies in your project's `pom.xml`
+
+```xml
+<dependency>
+  <groupId>com.feedhenry</groupId>
+  <artifactId>fh-android-sdk</artifactId>
+  <version>3.3.0</version>
+  <scope>provided</scope>
+  <type>jar</type>
+</dependency>
+
+<dependency>
+  <groupId>com.feedhenry</groupId>
+  <artifactId>fh-android-sdk</artifactId>
+  <version>3.3.0</version>
+  <type>aar</type>
+</dependency>
 ```
 
-This will compile the source code, generating an aar and a jar file in the _target_ directory.
+## Documentation
 
-## Usage
-
-### Gradle Projects (Recommended)
-
-Add the FH Android `aar` library in your `build.gradle` project
-
-```
-dependencies {
-	compile 'com.feedhenry:fh-android-sdk:3.1.0'
-}
-```
+For more details about that please consult [FH Android SDK Guide](http://docs.feedhenry.com/v3/dev_tools/sdks/android.html).
```

Comment:
This link redirects to https://access.redhat.com/documentation/en/red-hat-mobile-application-platform-hosted/, so as I commented before, maybe use this direct link to Android SDK instead  https://access.redhat.com/documentation/en-us/red_hat_mobile_application_platform/4.2/html/client_sdk/native-android